Trouble installing on ubuntu 16.04

I used this command, in bash, to install aggdraw for the entire system.

    sudo -H pip3 install aggdraw

The complete output follows:

    Collecting aggdraw
      Downloading https://files.pythonhosted.org/packages/42/33/2f6ba9ed2199181d1f2a6b346396134f98ff69a308f732acb36605bbc0b0/aggdraw-1.3.6.tar.gz (269kB)
    100% |████████████████████████████████| 276kB 7.5MB/s 
    Building wheels for collected packages: aggdraw
      Running setup.py bdist_wheel for aggdraw ... error
      Complete output from command /usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-fyrbgiq4/aggdraw/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /tmp/pip-wheel-tnafb16k --python-tag cp35:
      Trying freetype-config to find freetype library...
      Using ctypes to find freetype library...
      === freetype found: '/tmp/pip-install-fyrbgiq4'
      running bdist_wheel
      running build
      running build_ext
      building 'aggdraw' extension
      creating build
      creating build/temp.linux-x86_64-3.5
      creating build/temp.linux-x86_64-3.5/agg2
      creating build/temp.linux-x86_64-3.5/agg2/src
      creating build/temp.linux-x86_64-3.5/agg2/font_freetype
      x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -DVERSION=1.3.6 -DHAVE_FREETYPE2 -Iagg2/include -Iagg2/font_freetype -I/tmp/pip-install-fyrbgiq4/include -I/tmp/pip-install-fyrbgiq4/include/freetype -I/tmp/pip-install-fyrbgiq4/include/freetype2 -I/usr/include/python3.5m -c aggdraw.cxx -o build/temp.linux-x86_64-3.5/aggdraw.o
      cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++
      In file included from aggdraw.cxx:84:0:
      agg2/font_freetype/agg_font_freetype.h:23:22: fatal error: ft2build.h: No such file or directory
      compilation terminated.
      error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
  
      ----------------------------------------
      Failed building wheel for aggdraw
      Running setup.py clean for aggdraw
    Failed to build aggdraw
    Installing collected packages: aggdraw
      Running setup.py install for aggdraw ... error
        Complete output from command /usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-fyrbgiq4/aggdraw/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-hv7_mve1/install-record.txt --single-version-externally-managed --compile:
        Trying freetype-config to find freetype library...
        Using ctypes to find freetype library...
        === freetype found: '/tmp/pip-install-fyrbgiq4'
        running install
        running build
        running build_ext
        building 'aggdraw' extension
        creating build
        creating build/temp.linux-x86_64-3.5
        creating build/temp.linux-x86_64-3.5/agg2
        creating build/temp.linux-x86_64-3.5/agg2/src
        creating build/temp.linux-x86_64-3.5/agg2/font_freetype
        x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -DVERSION=1.3.6 -DHAVE_FREETYPE2 -Iagg2/include -Iagg2/font_freetype -I/tmp/pip-install-fyrbgiq4/include -I/tmp/pip-install-fyrbgiq4/include/freetype -I/tmp/pip-install-fyrbgiq4/include/freetype2 -I/usr/include/python3.5m -c aggdraw.cxx -o build/temp.linux-x86_64-3.5/aggdraw.o
        cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++
        In file included from aggdraw.cxx:84:0:
        agg2/font_freetype/agg_font_freetype.h:23:22: fatal error: ft2build.h: No such file or directory
        compilation terminated.
        error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
    
        ----------------------------------------
    Command "/usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-fyrbgiq4/aggdraw/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-hv7_mve1/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-install-fyrbgiq4/aggdraw/`

I found that I have 3 files named ft2build.h, but all are within other software's directories (docker and fsl) where pip would have no business looking. Do I need to install freetype source files before installing aggdraw? If so, can this be made an explicity dependency in aggdraw so it just works? Thanks for any help in getting this working the "right" way.

--- a/setup.py
+++ b/setup.py
@@ -61,6 +61,9 @@ def _get_freetype_with_ctypes():
             lib_path = os.path.join(bdir, 'lib', ft_lib_path)
             if os.path.isfile(lib_path):
                 return bdir
+        else:
+            # freetype is somewhere on the system, but we don't know where
+            return None
     ft_lib_path = os.path.dirname(ft_lib_path)
     lib_path = os.path.realpath(os.path.join(ft_lib_path, '..'))
     return lib_path

